decider-HSMetrics/src/test/resources/expected_output/development/StudyTest.json

```diff
@@ -0,0 +1,14 @@
+{
+  "workflowRunCount" : 0,
```

Comment:
No workflow runs were scheduled for "study test"

Reply:
Fixed